modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTables.scala

```diff
@@ -208,7 +209,15 @@ private sealed trait SequenceTablesBuilder[S: Eq, D: Eq]:
                 <.div(
                   <.div(
                     ObserveStyles.BreakpointHandle,
-                    stepId.map(^.onClick --> props.flipBreakpoint(_)).whenDefined
+                    stepId
```

Comment:
Do we wait for the server to indicate the breakpont is enabled or is it optimistically set on the client?

Reply:
For the moment, it is optimistically set. However, if the invocation results in an error, the UI will enter resync mode.

Anyway, maybe we should check for the server's response and (un)set the breakpoint again according the server's value. I will do that.